scripts/run-e2e-tests.sh

```diff
@@ -101,12 +101,15 @@ function install_kind() {
 }
 
 function install_spark() {
-  check_cmd "wget"
-  wget -qO-  https://archive.apache.org/dist/spark/spark-3.3.1/spark-3.3.1-bin-hadoop3.tgz | tar xzvf - && chmod +x spark-3.3.1-bin-hadoop3 && sudo mv spark-3.3.1-bin-hadoop3 /opt/.
-  exit_on_error "install spark failed."
-  sudo ln -s /opt/spark-3.3.1-bin-hadoop3 /opt/spark
-  exit_on_error "install spark failed. unable to create symlink"
-  export SPARK_HOME=/opt/spark/
+  PWD=$(pwd)
+  if [ ! -d "${PWD}/_spark" ]; then
```

Comment:
This should probably check for the whole directory we want to use (i.e. _spark/spark-3.3.1-bin-hadoop3), and if not, start with `rm -rf "${PWD}_spark"`. This way we can transparently upgrade in the future.